docs/admin-manual/configuration-macros.rst

```diff
@@ -6532,22 +6508,6 @@ do not specify their own with:
     ``DEFAULT_RANK`` and ``DEFAULT_RANK_VANILLA`` are defined, the value
     for ``DEFAULT_RANK_VANILLA`` is used for vanilla universe jobs.
 
-:macro-def:`DEFAULT_IO_BUFFER_SIZE`
-    HTCondor keeps a buffer of recently-used data for each file an
-    application opens. This macro specifies the default maximum number
-    of bytes to be buffered for each open file at the executing machine.
-    The *condor_status* ``buffer_size`` command will override this
-    default. If this macro is undefined, a default size of 512 KB will
-    be used.
-
-:macro-def:`DEFAULT_IO_BUFFER_BLOCK_SIZE`
-    When buffering is enabled, HTCondor will attempt to consolidate
-    small read and write operations into large blocks. This macro
-    specifies the default block size HTCondor will use. The
-    *condor_status* ``buffer_block_size`` command will override this
-    default. If this macro is undefined, a default size of 32 KB will be
-    used.
-
```

Comment:
A quick git grep shows that these are still being used in submit_utils.cpp

Reply:
Yes, but the classad attributes aren't being used for anything.  Eventually we'll rip that code out, but for now, no need to confuse user with extra info they don't need.